src/backend/distributed/citus.control

```diff
@@ -1,6 +1,6 @@
 # Citus extension
 comment = 'Citus distributed database'
-default_version = '10.2-4'
+default_version = '10.2-5'
```

Comment:
This creates a new upgrade path that would then also need a path to 11.0. We would need to be super careful that you arrive at the same schema regardless of which path you follow. We also need to be careful about the upgrade path finding logic skipping this file, because it prefers the shortest path.

Do we need to backport to 10.2? If so, I wonder whether it would be preferable to patch the old function definition. That does create two different variants of 10.2-4, which is also not ideal.

Reply:
> We would need to be super careful that you arrive at the same schema regardless of which path you follow.

I agree. I guess we can use copy the downgrade script and use that on release-11.0 and main branches. This way the changes will be reverted when upgrading from 10.2-5 to later major versions.

> Do we need to backport to 10.2?

Onder likes to backport this fix to all affected versions that support PG14 so that they do not run into issues when upgrading PG versions.

> If so, I wonder whether it would be preferable to patch the old function definition. That does create two different variants of 10.2-4, which is also not ideal.

I also think that it is not ideal, and I would not want to have different variants of a migration version.

Reply:
We currently don't install downgrade scripts, though we probably should.

Reply:
I agree that it is a good idea that we should start installing downgrade scripts by default. This architecture matured enough, and provided some benefits to us already. I think we may consider adding better checks around downgrade scripts before installing them by default. Having several alternative paths may have unintended consequences. I know PG will pick the shortest path available, but it can possibly be a path that is untested and buggy and we would not want that to happen.

We do not install all downgrade scripts but only some. For example https://github.com/citusdata/citus/pull/5076 placed some downgrade scripts on `sql` folder intentionally and we shipped those in the packages. I expect @agedemenli to create some similar files on `main` and `release-11.0` so that we have valid upgrade paths from `10.2-5`. Although some steps will be downgrades, the overall result of `ALTER EXTENSION citus UPDATE` will be a upgrade. We will also have a single path so that's good.